src/prefer-array-from.test.ts

```diff
@@ -0,0 +1,27 @@
+import { RuleTester } from "eslint";
+import { preferArrayFromRule } from "./prefer-array-from";
+
+const tester = new RuleTester({
+	parser: require.resolve("@typescript-eslint/parser"),
+	parserOptions: {
+		ecmaFeatures: {
+			jsx: true,
+		},
+	},
+});
+
+tester.run("prefer-array-from", preferArrayFromRule, {
+	valid: [],
```

Comment:
Do you mind including some valid cases here as well? It helps with making sure we don't error on good syntax

Reply:
Do you have any suggestions what good syntax would be here?